worker-db-ddl.sql

```diff
@@ -38,6 +47,585 @@ CREATE TYPE cqc."WorkerDisability" AS ENUM (
 	'Don''t know'
 );
 
+CREATE TYPE cqc."WorkerNationality" AS ENUM (
+	'British',
+	'Other',
+	'Don''t know'
+);
+
+CREATE TYPE cqc."WorkerCountryOfBirth" AS ENUM (
+	'United Kingdom',
+	'Other',
+	'Don''t know'
+);
+
+CREATE TYPE cqc."WorkerRecruitedFrom" AS ENUM (
+	'Yes',
+	'No'
+);
+
+-- Ethnicity Reference Data
+CREATE TABLE IF NOT EXISTS cqc."Ethnicity" (
+	"ID" INTEGER NOT NULL PRIMARY KEY,
+	"seq" INTEGER NOT NULL, 	-- this is the order in which the Ethinicity will appear without impacting on primary key (existing foreign keys)
+	"EthnicityGroup" TEXT NOT NULL,
+	"Ethnicity" TEXT NOT NULL
+);
+INSERT INTO cqc."Ethnicity" ("ID", "seq", "EthnicityGroup", "Ethnicity") VALUES 
+	(1, 1, '', 'English / Welsh / Scottish / Northern Irish / British'),
+	(2, 2, '', 'Don''t know'),
+	(3, 200, 'White', 'Irish'),
+	(4, 201, 'White', 'Gypsy or Irish Traveller'),
+	(5, 202, 'White', 'Any other White background'),
+	(6, 300, 'Mixed / multiple ethnic groups', 'White and Black Caribbean'),
+	(7, 301, 'Mixed / multiple ethnic groups', 'White and Black African'),
+	(8, 302, 'Mixed / multiple ethnic groups', 'White and Asian'),
+	(9, 303, 'Mixed / multiple ethnic groups', 'Any other Mixed/Multiple ethnic background'),
+	(10, 400, 'Asian / Asian British', 'Indian'),
+	(11, 401, 'Asian / Asian British', 'Pakistani'),
+	(12, 402, 'Asian / Asian British', 'Bangladeshi'),
+	(13, 403, 'Asian / Asian British', 'Chinese'),
+	(14, 404, 'Asian / Asian British', 'Any other Asian background'),
+	(15, 500, 'Black / African / Caribbean / Black British', 'African'),
+	(16, 501, 'Black / African / Caribbean / Black British', 'Caribbean'),
+	(17, 502, 'Black / African / Caribbean / Black British', 'Any other Black / African / Caribbean background'),
+	(18, 600, 'Other ethnic group', 'Arab'),
+	(19, 601, 'Other ethnic group', 'Any other ethnic group');
+
+-- Nationality Reference Data
+CREATE TABLE IF NOT EXISTS cqc."Nationality" (
+	"ID" INTEGER NOT NULL PRIMARY KEY,
+	"seq" INTEGER NOT NULL, 	-- this is the order in which the Ethinicity will appear without impacting on primary key (existing foreign keys)
```

Comment:
Hi Warren
The Nationality table, columns seq and nationality can be "Seq" and "Nationality", though this isn't affecting functionality anyway however we need to follow the format, what you advise on this?
Thanks
Shakir

Reply:
Good catch Shakir. I have updated the schema here, and updated the backend code to match the revised column names, and have retested all of the five reference APIs that consume these tables.

If you happy with the revised schema and feedback to this comment, please approve the PR and I'll look to merging down to master, and effecting the changes on the DEV database instance.